extensions/mssql/src/objectManagement/interfaces.ts

```diff
@@ -546,8 +550,15 @@ export interface ServerViewInfo extends ObjectManagement.ObjectViewInfo<Server>
 }
 
 export interface DatabaseFile {
+	id: number;
 	name: string;
 	type: string;
 	path: string;
 	fileGroup: string;
+	fileNameWithExtension: string;
+	sizeInMb: number;
+	isAutoGrowthEnabled: boolean;
+	autoFileGrowth: number;
+	autoFileGrowthType: string;
```

Comment:
Can this be an enum value instead to make it easier to work with?

Reply:
Yeah that would be very helpful, working on changes!